specificatie/reisdocument.yaml

```diff
@@ -6,90 +6,102 @@ info:
 paths: {}
 components:
   schemas:
-    Reisdocumentnummer:
-      type: string
-      pattern: ^[a-zA-Z0-9]{9}$
-    Burgerservicenummer:
-      type: string
-      pattern: ^[0-9]{9}$
-      example: "555555021"
-    AanduidingInhoudingOfVermissing:
-      type: string
-      pattern: ^(I|R|V|\.)$
-    SoortReisdocument:
-      type: string
-      description: |
-            Een code voor het soort reisdocument. Wordt gevuld met een code die voorkomen in de tabel 'Nederlands_Reisdocument' uit de Haal-Centraal-BRP-tabellen-bevragen API.
-      pattern: ^[A-Z]{2}$
-    OpschortingBijhouding:
+    GbaInhoudingOfVermissing:
       type: object
       properties:
-        reden:
+        datumInhoudingOfVermissing:
+          $ref: 'datum.yaml#/components/schemas/GbaDatum'
+        aanduidingInhoudingOfVermissing:
           $ref: 'common.yaml#/components/schemas/Waardetabel'
-    Reisdocument:
+    InhoudingOfVermissing:
       type: object
-      description: |
-                    Een document dat nodig is voor reizen naar het buitenland.
-
-                    indicatieGeldig geeft aan of het reisdocument op dit moment nog geldig en in het bezit van de persoon is. Wordt gevuld op basis van de waarden die voorkomen in de tabel 'Geldig_Reisdocument' uit de Haal-Centraal-BRP-tabellen-bevragen API.
       properties:
-        reisdocumentnummer:
-          $ref: '#/components/schemas/Reisdocumentnummer' 
-        indicatieGeldig:
+        datumInhoudingOfVermissing:
+          $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+        aanduidingInhoudingOfVermissing:
           $ref: 'common.yaml#/components/schemas/Waardetabel'
+        inOnderzoek:
+          $ref: '#/components/schemas/InhoudingOfVermissingInOnderzoek'
+    InhoudingOfVermissingInOnderzoek:
+      type: object
+      properties:
+        datumInhoudingOfVermissing:
+          type: boolean
+        aanduidingInhoudingOfVermissing:
+          type: boolean
+        datumIngangOnderzoek:
+          $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+    ReisdocumenthouderBasis:
+      type: object
+      properties:
         burgerservicenummer:
-          $ref: '#/components/schemas/Burgerservicenummer'
-    ReisdocumentUitgebreid:
+          $ref: 'persoon.yaml#/components/schemas/Burgerservicenummer'
+        geheimhoudingPersoonsgegevens:
```

Comment:
deze wordt nu gebruikt in GbaReisdocumenthouder en in Reisdocumenthouder. In de gba versie moet dit een getal zijn (GbaGeheimhoudingPersoonsgegevens), in de proxy versie moet dit een boolean zijn (GeheimhoudingPersoonsgegevens)

Reply:
aangepast